File: java/src/test/java/mx/iteso/ut/TortillaMaizTest.java
package mx.iteso.ut;

import org.junit.Test;
import org.junit.Before;
import static org.junit.Assert.*;


public class TortillaMaizTest {
    TortillaMaiz tortillaMaiz;

    @Before
    public void setup(){
        tortillaMaiz = new TortillaMaiz();
    }

    @Test
    public void TestCurrentTemperature(){
        tortillaMaiz.setCurrentTemperature(20);
        assertEquals(20,tortillaMaiz.getCurrentTemperature());
    }

    @Test
    public void TestTrueToasted(){
        tortillaMaiz.toast(true);
        assertTrue(tortillaMaiz.isToasted());
    }

    @Test
    public void TestFalseToasted(){
        tortillaMaiz.toast(false);
        assertFalse(tortillaMaiz.isToasted());
    }

    @Test
    public void TestToasting(){

        assertEquals(10, tortillaMaiz.getToastTemperature());
    }
}

File: python/src/unittest/python/TortillaMaizTest.py
from TortillaMaiz import *
from mockito import when, mock, verify,assertEquals,assertFalse,assertTrue
import unittest
class TortillaMaizTest(unittest.TestCase):
    def __init__(self):
        self.tortillaMaiz = TortillaMaiz()

    def testCurrentTemperature(self):
        self.tortillaMaiz.setCurrentTemperature(11)
        assertEquals(11,self.tortillaMaiz.getCurrentTemperature())

    def testFalseToast(self):
        self.tortillaMaiz.toast(False)
        assertFalse(self.tortillaMaiz.isToasted())

    def testTrueToast(self):
        self.tortillaMaiz.toast(True)
        assertTrue(self.tortillaMaiz.isToasted())

    def testToasting(self):
        assertEquals(10,self.tortillaMaiz.getToastTemperature())